modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/Translation.js

```diff
@@ -86,23 +86,13 @@ export default function Translation({
 									<ClayLayout.ContentCol expand>
 										<span>{language.languageId}</span>
 									</ClayLayout.ContentCol>
-									<ClayLayout.ContentCol>
-										<ClayLabel
-											displayType={
-												language.default
-													? 'primary'
-													: 'success'
-											}
-										>
-											{language.default
-												? Liferay.Language.get(
-														'default'
-												  )
-												: Liferay.Language.get(
-														'translated'
-												  )}
+									{language.default ? (
```

Comment:
You don't need a ternary here 😄   You can just do:

```js
{language.default && (
	<ClayLabel displayType="primary">
		{Liferay.Language.get('default')}
	</ClayLabel>
)}
```

Reply:
Ohhh!!!, i like this approach :). Awesome!!

Reply:
I have applied your feedback in https://github.com/liferay-tango/liferay-portal/pull/959/commits/f17cc40c243cc69cf7264be1770facf82b371b83